specification/ai/Azure.AI.Agents/tools/models.tsp

```diff
@@ -782,6 +822,30 @@ model ConnectedAgentDetails {
   description: string;
 }
 
+/**
+ * The details of the Deep Research tool.
+ */
+@added(Versions.v2025_05_15_preview)
+model DeepResearchDetails {
+  @doc("The deep research model deployment name.")
+  @encodedName("application/json", "deep_research_model")
+  `model`: string;
+
+  @doc("The array containing Bing grounding connection IDs to enhance deep research capabilities.")
+  @encodedName("application/json", "bing_grounding_connections")
+  deepResearchBingGroundingConnections: DeepResearchBingGroundingConnection[];
```

Comment:
Similar question here - why are we repeating `deepResearch` throughout the hierarchy. This is mostly a question for the REST API design though?

Reply:
Renamed.